plugins/woocommerce-admin/client/launch-your-store/hub/sidebar/components/launch-store-hub.tsx

```diff
@@ -1,40 +1,176 @@
+							type: 'OPEN_WC_ADMIN_URL',
+							url: 'admin.php?page=wc-settings&tab=general',
+						} );
+					} }
+					href=""
+				/>
+			),
+		}
+	);
+
+	const hasIncompleteTasks =
+		tasklist && ! tasklist.tasks.every( ( task ) => task.isComplete );
+
+	const [ removeTestOrders, setRemoveTestOrder ] = useState(
+		removeTestOrdersContext ?? true
+	);
+
 	return (
 		<div
 			className={ classnames(
 				'launch-store-sidebar__container',
 				props.className
 			) }
 		>
-			<p>Sidebar</p>
-			<button
-				onClick={ () => {
-					props.sendEventToSidebar( { type: 'LAUNCH_STORE' } );
-					// when we send LAUNCH_STORE to the sidebar machine, the sidebar machine sends the appropriate event to the main content machine to show the launching state
-				} }
+			<motion.div
+				className="edit-site-layout__header-container"
+				animate={ 'view' }
 			>
-				Launch Store
-			</button>
-			<button
-				onClick={ () => {
-					props.sendEventToSidebar( {
-						type: 'OPEN_EXTERNAL_URL',
-						url: 'https://example.com',
-					} );
-				} }
+				<SiteHub
+					as={ motion.div }
+					variants={ {
+						view: { x: 0 },
+					} }
+					isTransparent={ false }
+					className="edit-site-layout__hub"
+				/>
+			</motion.div>
+			<SidebarContainer
+				title={ sidebarTitle }
+				description={ sidebarDescription }
 			>
-				Open external URL
-			</button>
+				<div className="edit-site-sidebar-navigation-screen-essential-tasks__group-header">
+					<Heading level={ 2 }>
+						{ __( 'Essential Tasks', 'woocommerce' ) }
+					</Heading>
+				</div>
+				<ItemGroup className="edit-site-sidebar-navigation-screen-essential-tasks__group">
+					{ tasklist &&
+						hasIncompleteTasks &&
+						tasklist.tasks.map( ( task ) => (
+							<SidebarNavigationItem
+								className={ classnames( task.id, {
+									'is-complete': task.isComplete,
+								} ) }
+								icon={
+									task.isComplete
+										? taskCompleteIcon
+										: taskIcons[ task.id ]
+								}
+								withChevron
+								key={ task.id }
+								onClick={ () => {
+									props.sendEventToSidebar( {
+										type: 'TASK_CLICKED',
+										task,
+									} );
+								} }
+							>
+								{ task.title }
+							</SidebarNavigationItem>
+						) ) }
+					{ tasklist && ! hasIncompleteTasks && (
+						<SidebarNavigationItem
+							className="all-tasks-complete"
+							icon={ taskCompleteIcon }
+						>
+							{ __(
+								'Fantastic job! Your store is ready to go — no pending tasks to complete.',
+								'woocommerce'
+							) }
+						</SidebarNavigationItem>
+					) }
+				</ItemGroup>
+				<div className="edit-site-sidebar-navigation-screen-test-data__group-header">
+					<Heading level={ 2 }>
+						{ __( 'Test data', 'woocommerce' ) }
+					</Heading>
+				</div>
+				<ItemGroup className="edit-site-sidebar-navigation-screen-remove-test-data__group">
+					<ToggleControl
+						label={ __( 'Remove 20 test orders', 'woocommerce' ) }
```

Comment:
I assume `20` here is supposed to be dynamic, but perhaps this is going to be done in a follow-up?

I'd suggest to still use something like `sprintf` for crafting the string here

Reply:
I also wonder now what happens if there's 0 test order? Should we remove the toggle altogether? 🤔

Reply:
Ah sorry, we won't launch MVP with test order functionality. We'll follow-up with that shortly after.

Reply:
> I assume 20 here is supposed to be dynamic, but perhaps this is going to be done in a follow-up?

I knew I forgot something... I filed it away to follow up but I didn't write it on my todo list. Might need to write a todo list app to help with that.

Thanks for catching this!!

> Ah sorry, we won't launch MVP with test order functionality. We'll follow-up with that shortly after

Ok I'll just make this section dependent on how many test orders were found, which we will default-populate with 0 for now